recipes-tag/bluesky-kafka/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "0.1" %}
+
+package:
+    name: bluesky-kafka
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/nsls-ii/bluesky-kafka
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    skip: True  # [py2k]
+
+requirements:
+    build:
+        - python
+
+    run:
+        - bluesky
+        - pykafka
+
+test:
+    imports:
+        - bluesky
```

Comment:
Do you also want to test if pykafka is imported properly?

Reply:
i meant to change that to bluesky_kafka oops thanks!

Reply:
I think we still need this update :)

Reply:
oops forgot to push thanks!